assets/css/less/wpbdp/base/base.less

```diff
@@ -0,0 +1,46 @@
+.wpbdp-page,
+.wpbdp-single,
+.wpbdp-excerpt {
+
+	&,
+	& * {
+		box-sizing: border-box;
+	}
+
+	p {
+		margin-top: 0;
+		margin-bottom: 20px;
+		color: @color-text--60;
```

Comment:
I'm afraid this will make the directory not match the rest of the site. Let's be very considerate about what we use from the theme, and what we customize. I don't think we should change the color, font-size, font-weight, or line height here, or in links.

Reply:
Ok, I'll remove them.